utils/.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit=
+    */lib/python*/site-packages/*
```

Comment:
@rhcarvalho The main reason is, because removing the package filter causes additional code to be pulled in for the coverage report, including packages installed in the virtualenv, and in some cases stdlib packages. The omit config removes these items from the coverage report.